docs/decisions/0004-downstream-effect-events.rst

```diff
@@ -0,0 +1,103 @@
+Implementation of Event Driven Architecture for Exam Downstream Effects
+=======================================================================
+
+Status
+------
+
+Draft (circa Sept 2023)
+
+Context
+-------
+For Milestone 6 of the exams IDA project, we are porting any downstream effects of exam submission and review
+from edx-proctoring to the exams IDA (aka edx-exams). Most of these downstream effects will reflect calls
+that edx-proctoring (which is a plugin) makes to various edx-platform services, such as grades, credits, and certificates.
+
+Since edx-exams is independent, we plan to use the event bus to send info to edx-platform services
+without needing a response as one would in a REST setup.
+
+This is being done to circumvent messy circular dependencies, wherein edx-exams and edx-platform go back and
+forth asking each other for information.
+
+
+Decision
+--------
+We are porting these because they are both essential and easy to translate into events:
```

Comment:
Can you be more specific about what "these" refers to? "These" refers to "downstream effects of exam submission and review", right? It would be useful to tie those two paragraphs together.